hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/DNCertificateClient.java

```diff
@@ -35,14 +51,26 @@ public class DNCertificateClient extends DefaultCertificateClient {
       LoggerFactory.getLogger(DNCertificateClient.class);
 
   public static final String COMPONENT_NAME = "dn";
+  private final DatanodeDetails dn;
+  private SCMSecurityProtocolClientSideTranslatorPB secureScmClient;
 
-  public DNCertificateClient(SecurityConfig securityConfig,
-      String certSerialId) {
-    super(securityConfig, LOG, certSerialId, COMPONENT_NAME);
+  public DNCertificateClient(OzoneConfiguration ozoneConfig,
```

Comment:
I am not sure what is the advantage of changing here to OzoneConfiguration from SecurityConfig, can you share some reasoning for this change please?